tests/test_route.py

```diff
@@ -623,3 +585,94 @@ def test_RouteAndNexthopInDifferentVrf(self, dvs, testlog):
         dvs.servers[2].runcmd("ip address del 20.0.0.2/24 dev eth0")
         dvs.servers[3].runcmd("ip route del default dev eth0")
         dvs.servers[3].runcmd("ip address del 20.0.1.2/24 dev eth0")
+
+class TestRoutePerf(TestRouteBase):
+    """ Performance tests for route """
+    def test_PerfAddRemoveRoute(self, dvs, testlog):
+        self.setup_db(dvs)
+        self.clear_srv_config(dvs)
+        numRoutes = 10000   # number of routes to add/remove
+
+        # generate addresses of routes
+        addrs = []
+        for i in range(numRoutes):
+            addrs.append("%d.%d.%d.%d/%d" % (100 + int(i / 256 ** 2), int(i / 256), i % 256, 0, 24))
```

Comment:
>addrs [](start = 12, length = 5)

they are not ip addresses, but ip prefixes.

Reply:
Fixed.